tests/functional/ext/docparams.py

```diff
@@ -16,7 +16,7 @@ def _private_func3(param1):  # [missing-raises-doc]
     raise Exception('Example')
 
 
-def public_func1(param1):  # [missing-param-doc, missing-type-doc]
+def public_func1(param1):  # [missing-any-param-doc]
```

Comment:
As we can't use old names we should warn in whats new for 2.12 that this can happen.